packages/circle-react-elements/src/components/ui/toast.tsx

```diff
@@ -0,0 +1,127 @@
+import * as ToastPrimitives from '@radix-ui/react-toast';
+import { cva, type VariantProps } from 'class-variance-authority';
+import { X } from 'lucide-react';
+import * as React from 'react';
+
+import { cn } from '../../lib/utils';
+
+const ToastProvider = ToastPrimitives.Provider;
```

Comment:
To keep the new package clean, let’s only move components from the /ui/ folder that are actually used by the Circle components we’re migrating here. For example, we definitely don’t need toast/useToast, pagination, and likely even card and dialog, as these components are only used in the demo app as wrappers for Circle components.

Reply:
fixed